### DISCUSSION TITLE
Expected to Immediate Working

### DISCUSSION THREAD

**Discussion by unmeimusu** [importantimport/shiraha#8] [Q&A]
I've tried to add Shiraha classless CSS file to my website, but it doesn't work. please check, thanks
https://cc7z.blogspot.com/

**Comment by kwaa** [comment]
Shiraha is not yet ready to use.

In addition, your blogger theme contains inline styles and deep nesting, which will affect the display to some extent.

This use case may be optimized later (after 0.1), but for now I'll focus on simple and semantic HTML.

> Note: Shiraha does not come with a theme, you need to import additional themes.
>
> `<link href="https://unpkg.com/shiraha/dist/themes/baseline.css" rel="stylesheet">`

Also see:

- [Example of HTML in the Documentation](https://shiraha.js.org/?path=%2Fstory%2Fcomponents-header--medium-header)

- [Lume Theme Shiraha](https://github.com/importantimport/lume_theme_shiraha)
